test/test_google_geocoder.rb

```diff
@@ -190,6 +190,24 @@ def test_google_suggested_bounds_url
     Geokit::Geocoders::GoogleGeocoder.geocode("Winnetka", bias: bounds)
   end
 
+  def test_google_place_id
+    VCR.use_cassette("google_full_v3_20") do
+      url = "https://maps.google.com/maps/api/geocode/json?sensor=false&address=#{Geokit::Inflector.url_escape(@full_address_short_zip)}"
+      TestHelper.expects(:last_url).with(url)
+      res = Geokit::Geocoders::GoogleGeocoder.geocode(@full_address_short_zip)
+      assert_equal 'EjExMDAgU3BlYXIgU3RyZWV0ICM1LCBTYW4gRnJhbmNpc2NvLCBDQSA5NDEwNSwgVVNB', res.place_id
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.